api/open_api_3.yaml

```diff
@@ -170,28 +179,27 @@ paths:
           $ref: '#/components/responses/StatusNotFound'
         '500':
           $ref: '#/components/responses/StatusInternalServerError'   
-  /list/{peername}:
-    parameters:
-      - name: peername
-        in: path
-        description: Peername
+  /save:
+    post:
+      summary: Save and update to a dataset head
+      operationId: saveDatasetFromRoot
+      requestBody:
+        description: Updated dataset head
         required: true
-        schema:
-          type: string
-    get:
-      summary: Get the list a specific peer’s datasets
-      operationId: getPeersDatasetList
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Dataset'       
       responses:
         '200':
-          $ref: '#/components/responses/DatasetsResponse'
+          $ref: '#/components/responses/DatasetResponse'
         '403':
           $ref: '#/components/responses/StatusForbidden'
         '404':
           $ref: '#/components/responses/StatusNotFound'
         '500':
-          $ref: '#/components/responses/StatusInternalServerError'  
-  /save:
-    post:
+          $ref: '#/components/responses/StatusInternalServerError'
+    put:
```

Comment:
@b5 another quick question here -- I noticed that we often support multiple HTTP verbs for an endpoint (like POST and PUT here, or DELETE and POST for /remove) but it seems like we tend to only document the former (I'd assume more common?) case. I added in this documentation for PUT /save but should I remove? Trying to figure out how thorough we'd like to be here